assets/js/base/components/store-notices-container/style.scss

```diff
@@ -24,3 +24,88 @@
 		margin-top: 1em;
 	}
 }
+
+/** Snackbar notices styles **/
+.components-snackbar {
```

Comment:
Is this styling for (cart/checkout) notice snackbars, or all snackbars anywhere? Should probably have a class targeting these styles to `store-notices-container`, or factor out our own woo frontend snackbar component (if this is intended to be used for things other than notices).

Also, do we get any styling for free from the `SnackbarList` component? It looks like we're reinventing snackbar styling here.

Reply:
> Is this styling for (cart/checkout) notice snackbars, or all snackbars anywhere?

This isn't a component specific to cart/checkout, so everywhere (in the frontend).

> Also, do we get any styling for free from the SnackbarList component? It looks like we're reinventing snackbar styling here.

Ya I thought we'd get that styling for free with our current webpack config but it doesn't look like we are.  I'm guessing because gutenberg doesn't directly import the styles but has them pulled in via build process instead (so our webpack config isn't picking them up).  

I definitely would rather pull in the styles from the package. @Aljullu do you have any ideas around this?

Reply:
@nerrad you need to specify the component's CSS file in the webpack entry config:

https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/master/bin/webpack-helpers.js#L107-L114

Reply:
Thanks for the tip Albert, that did the trick 🌮